Pop Alexia/reverse/main.c

```diff
@@ -0,0 +1,33 @@
+#include <stdio.h>
+#include <stdlib.h>
+
+/*Your goal is to read a 68-word text from the input and then print it to the screen backwards.
+ Individual words do not have to be spelled backwards, but rather your program should print out
+ the last word first, then the second-to-last word, etc. No word has more than 40 characters.
+ Example
+Input
+Science Computer on Papers Selected Knuth, Ervin Donald ― correct." be will results the that
+reader a convince to and works algorithm an way the communicate to concepts, mathematical as
+well as forms literary and aesthetic traditional with works who essayist an ideally is programmer
+A clearly. them understand can beings human that so and quickly them perform can machines computing
+ that so written are programs best "The
+
+Output
+"The best programs are written so that computing machines can perform them quickly and so that human
+beings can understand them clearly. A programmer is ideally an essayist who works with traditional
+aesthetic and literary forms as well as mathematical concepts, to communicate the way an algorithm works
+and to convince a reader that the results will be correct." ― Donald Ervin Knuth, Selected Papers on Computer Science*/
+
+int main(void)
+{
+ char text[68][41];
+ int i;
+for(i=0;i<68;i++);
```

Comment:
Da, uite aici e problema
Din păcate, la finalul rândului cu `for`, ai un `;` care de fapt face ca acel `for` să nu facă absolut nimic (practic iterează în gol)

Dacă îl ștergi, va merge.

Recomandarea mea totuși este să nu citești de la consolă 60+ cuvinte!!!! ci să le citești măcar dintr-un fișier, altfel pierzi mult prea mult timp cu asta.